.gitignore

```diff
@@ -1,23 +1,14 @@
 # intentionally ignore
-public/data
+vendor/
+public/data/umpr_repos/
+app/logs/
 
-*.class
+composer.lock
 
-# Mobile Tools for Java (J2ME)
-.mtj.tmp/
-
-# Package Files #
-*.jar
-*.war
-*.ear
-
-# WebStorm
+# *Stork products
 .idea/
 
-# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
-hs_err_pid*
-
 .DS_Store
 
 # Go find your own umple.jar
-umple.jar
+umple.jar
```

Comment:
I personally like newlines at the end of files so can I more easily "cat" it. It can be automatically in my sublime preferences

```json
    // Set to true to ensure the last line of the file ends in a newline
    // character when saving
    "ensure_newline_at_eof_on_save": false,
```